vpr/src/draw/draw.cpp

```diff
@@ -1899,10 +1899,10 @@ static void draw_chanx_to_chanx_edge(int from_node, int to_node, int to_track, s
     y1 = from_chan.bottom();
     y2 = to_chan.bottom();
 
-    from_xlow = device_ctx.rr_nodes[from_node].xlow();
-    from_xhigh = device_ctx.rr_nodes[from_node].xhigh();
-    to_xlow = device_ctx.rr_nodes[to_node].xlow();
-    to_xhigh = device_ctx.rr_nodes[to_node].xhigh();
+    from_xlow = rr_graph.node_xlow(RRNodeId(from_node));
```

Comment:
Same comment on from_node, to_node etc. -- could we just declare them to be of type RRNodeId and reduce casting?